### PR TITLE
Adding role and English file + optimizations

### PR DESCRIPTION
**Pull request by Romitou**
This Pull Request brings some modifications and addition of a role made by TitiLaPierre.
An English language file has been added. Feel free to let me know about errors in this file.

### CODE REVIEW

File: Wolfo/lang/en_us.yml [hunk 1]
```diff
@@ -99,28 +96,25 @@ MESSAGE:
...
-        - |4-
-              &f» &bThe &6two lovers as well as the &6cupidon are the last three
-            survivor
+        - '&fHere are the conditions for victory:'
+        -  ''
+        -  '  &f» &bAll villages are dead.'
+        -  '  &f» &bAll werewolves are dead.'
+        -  '  &f» &6Crazy&b is killed by the village.'
+        -  '  &f» &bThe target of the headhunter is killed by the village.'
+        -  "|4-"
```
Comment (TitiLaPierre) [inline Wolfo/lang/en_us.yml:105]: The lovers's victory message was replaced with "|4-"

File: Wolfo/lang/en_us.yml [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+            - '&7you don''t have any &bfaculty&7.'
+            - ''
+            - '&8➥ &7Your team: &6Village'
+    CUPID:
+        NAME: '&dCupid'
+        DESC:
+            - ''
+            - '&8➥ &7At the beginning of the game, you must'
+            - '&7select two players who will be'
+            - '&7lover during the whole game. If one'
```
Comment (Matocolotoe) [inline Wolfo/lang/en_us.yml:189]:
```suggestion
            - '&7lovers during the whole game. If one'
```

File: Wolfo/lang/en_us.yml [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+            - ''
+            - '&8➥ &7Your team: &6Village'
+    WITCH:
+        NAME: '&5Witch'
+        DESC:
+            - ''
+            - '&8➥ &7You have &two potions'
+            - '&7One allows a player to succeed,'
+            - '&7the other allows you to kill a player. You'
+            - '&7can only be used once.'
```
Comment (Matocolotoe) [inline Wolfo/lang/en_us.yml:256]: "You can only be used once" ?

**Comment by Romitou** [comment]
Thanks!

**Comment by Romitou** [comment]
Will be done before the end of July.

**Comment by Romitou** [comment]
This Pull Request makes a lot of changes. If you want to test (clone the dev branch), don't hesitate to let me know if you encounter any problems!

**Pull request merged**